inspirehep/modules/pidstore/minters.py

```diff
@@ -41,5 +41,5 @@ def inspire_recid_minter(record_uuid, data):
     if 'control_number' in data:
         args['pid_value'] = data['control_number']
     provider = InspireRecordIdProvider.create(**args)
-    data['control_number'] = str(provider.pid.pid_value)
+    data['control_number'] = provider.pid.pid_value
```

Comment:
Ah-ah! With @rikirenz and @david-caro we just discovered that, if you omit line 4, the `pid_value` that is minted is actually a string.